queen_attack/queen_attack.rb

```diff
@@ -1,32 +1,33 @@
+# frozen_string_literal: true
+
+# Class Queens for chess.
 class Queens
-	def initialize(white = nil, black = nil)
-		begin
-	    @white = white
-	    @black = black
-	  raise ArgumentError, "You don't enter a position." if correct_value?	 
-	  rescue => mes
-	  	p mes
-	  end
+  def initialize(white = nil, black = nil)
+    @white = white
+    @black = black
+    raise ArgumentError, "You don't enter a position." if correct_value?
+  rescue StandardError => e
```

Comment:
Don't raise error when you can check in other way and return response with error message.